### PR TITLE
LPS-132769

### PR DESCRIPTION
**Pull request by lfbesada**

### CODE REVIEW

**Comment by lfbesada** [comment]
ci:test:sf

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:reevaluate:1127401_4884

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

**Comment by lfbesada** [comment]
ci:test:relevant

File: modules/apps/portal/portal-servlet-test/src/testIntegration/java/com/liferay/portal/servlet/test/I18nServletTest.java [hunk 1]
```diff
@@ -334,6 +336,45 @@ public void testSendRedirectWithoutContext() throws Exception {
 			mockHttpServletResponse.getHeader("Location"));
 	}
 
+	@Test
+	public void testServiceIfNotGroupFoundResponseForwardedToLocalized404Page()
```
Comment (adolfopa) [inline modules/apps/portal/portal-servlet-test/src/testIntegration/java/com/liferay/portal/servlet/test/I18nServletTest.java:340]: What about changing the test name to some like: `testResponseIsForwardedToLocalized404PageIfGroupNotFound`

File: portal-impl/src/com/liferay/portal/servlet/I18nServlet.java [hunk 2]
```diff
@@ -178,6 +182,8 @@ protected I18nData getI18nData(HttpServletRequest httpServletRequest)
 					friendlyURL);
 
 				if (siteGroup == null) {
+					httpServletRequest.setAttribute("I18nErrorPath", i18nPath);
```
Comment (adolfopa) [inline portal-impl/src/com/liferay/portal/servlet/I18nServlet.java:185]: Can you create a constant for "I18nErrorPath"? The name should be something like I18N_ERROR_PATH (and the value "I18N_ERROR_PATH"; same capitalization).

**Review by adolfopa — CHANGES_REQUESTED** [review]

**Pull request closed without merging**